.cirrus.yml

```diff
@@ -97,10 +97,14 @@ task:
   merge_script:
```

Comment:
I am also wondering if the cpu can be reduced to `2` (The whole project only has 16 parallel Cirrus CPU, so it is more efficient to run more tasks in parallel with less CPU, and thus less idle CPU)

Reply:
Are CPUs for Windows containers separated from ones for Linux containers?

Reply:
Also msvc cannot use `ccache`, and brute CPU force seems pretty welcome here :tiger2:

Reply:
WIP: https://github.com/ccache/ccache/pull/506